independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/common/providers/serialisers/InputStreamMessageBodyHandler.java

```diff
@@ -23,12 +23,20 @@ public InputStream readFrom(Class<InputStream> type, Type genericType, Annotatio
 
     @Override
     public boolean isWriteable(Class<?> type, Type genericType, Annotation[] annotations, MediaType mediaType) {
+        return doWriteTo(type);
+    }
+
+    protected boolean doWriteTo(Class<?> type) {
```

Comment:
Sorry for not being clear in the first time, but because of the triviality of this method, I think it is not needed and you can do `InputStream.isAssignableFrom(class)` in the subclasses. 

Sorry for nitpicking :)

Reply:
Should be a little better now :)